web/src/portico/tabbed-instructions.ts

```diff
@@ -31,25 +31,25 @@ export function activate_correct_tab($codeSection: JQuery<HTMLElement>): void {
     const $blocks = $codeSection.find(".blocks div");
 
     $li.each(function () {
-        const language = this.dataset.language;
+        const tab_key = this.dataset.tab_key;
```

Comment:
In Typescript, the `dataset` property is an object that provides access to custom data attributes on an HTML element. The properties in `dataset` are in camelCase rather than snake_case.

So the following would retrieve the value from the `data-tab-key` HTML data attribute from the templates in `tabbed_sections.py`:

```suggestion
        const tab_key = this.dataset.tabKey;
```